README.md

```diff
@@ -151,13 +171,45 @@ tests/:
 	checked into Git.
 ```
 
+## "Hello World" Application
```

Comment:
Is authorization discussed anywhere in this repo currently? Just to point out that authorization is omitted for simplicity?